package.json

```diff
@@ -25,8 +25,8 @@
   "dependencies": {
     "bluebird": "^3.0.6",
     "chalk": "^1.1.1",
-    "lodash": "^3.10.1",
```

Comment:
lodash is still being used within `src/createVariants.js` so we can't remove it completely. Might be feasible to import only the parts that we actually use, though.